{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/app.py

```diff
@@ -21,6 +22,7 @@ def db_manager_cls(self):
 
     def on_init_complete(self):
         extensions.auth_manager.init_app(self)
+        extensions.auth_manager.grid_cls = Grid
```

Comment:
Feel like just hanging this off an attribute isn't great.  Maybe something that should be passed into init_app()?

Reply:
You can leave this if you want, just pointing out it's unresolved.